pom.xml

```diff
@@ -390,7 +390,7 @@
         <configuration>
           <configLocation>${checkstyle.path}tachyon_checks.xml</configLocation>
           <includeTestSourceDirectory>true</includeTestSourceDirectory>
-          <excludes>**/tachyon/thrift/**</excludes>
+          <excludes>**/org/apache/jsp/**,**/tachyon/thrift/**</excludes>
```

Comment:
Is this the correct way to specify multiple excludes? I've seen other examples in pom files that list a new <exclude></exclude> per entry, but I don't know if that paradigm applies here.